doc/dao/Index.md

```diff
@@ -0,0 +1,44 @@
+# Contributing as part of the DAO
```

Comment:
Can this file be named README.md, to keep with top-level entry-point docs in the rest of the repo?